Automattic/jetpack
# VideoPress: set video privacy from Video Grid / dashboard

This PR implements the ability to set video privacy from the UI -> the Library Grid view.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: set video privacy from Video Grid / dashboard

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to VideoPress dashbord
* Change video privacy by clicking on the Privacy button fo the quick actions section
* Confirm the privacy changes on the fly
* Confirm the new privacy value persistent after a hard-refresh


https://user-images.githubusercontent.com/77539/192045754-8cf5fb32-0daa-4bad-823a-dd9cb8eb8f50.mov
